lib/simplecov/defaults.rb

```diff
@@ -47,7 +47,7 @@
   # If we are in a different process than called start, don't interfere.
   next if SimpleCov.pid != Process.pid
 
-  if $ERROR_INFO # was an exception thrown?
+  if $! # was an exception thrown?
```

Comment:
great, so rebase and merge?

I have a patch in my library now to add the alias :(